json-lint/schemas/inventory/adservers/taboola.json

```diff
@@ -68,6 +68,10 @@
       "enum": [
         "auto"
       ]
+    },
+    "disableMarfeelStyles": {
```

Comment:
if this attribute appears in the `<AMP-AD type=taboola>` isn't the AMP runtime going to complain?

Reply:
Nope, it prefixes data- in front of it

Reply:
In Gutenberg we are putting data- if the property is not reserved